README.md

```diff
@@ -55,3 +55,17 @@ When built with `SIGNING_SUPPORT` a [dynamic engine](https://github.com/openssl/
 When built with `MBEDTLS_SUPPORT` a dynamic library is produced providing an API to let MbedTLS key use the signing key. See the [header file](include/TobMbedtls.h) for the details.
 
 In a resource-constrained application you most likely don't want to use this library, and probably have your own way to connect to the modem/SIM. In this way you can statically link to the low-level library. The [API](include/TbMbedtlsLL.h) allows you to substitute your own implementation of the [interface to the modem](external_libs/tob_sim/common/inc/SEInterface.h). The library doesn't use dynamic memory or multithreading.
+
+# Sample Azure IoT Registration Client Helper
+
+`azure_dps_registerer` is a simple client that uses the Azure IoT C SDK to register a device using the Device Provisioning Service.
```

Comment:
Do we have a JIRA card detailing the changes we need for the Quest and Quickstart content to reflect the change from `python_tob_helper`?  Also, has the output format changed at all that will require a change to the python script that consumes the output?

Reply:
Not sure what was in the Quest, I'll recheck. The output format has changed to JSON, I'll update the README. The python sample is already updated (changes included in this PR).